packages/@stimulus/core/src/event_listener.ts

```diff
@@ -3,16 +3,18 @@ import { Binding } from "./binding"
 export class EventListener implements EventListenerObject {
   readonly eventTarget: EventTarget
   readonly eventName: string
+  readonly eventOptions: AddEventListenerOptions | false
   private unorderedBindings: Set<Binding>
 
-  constructor(eventTarget: EventTarget, eventName: string) {
+  constructor( eventTarget: EventTarget, eventName: string, eventOptions: AddEventListenerOptions) {
     this.eventTarget = eventTarget
     this.eventName = eventName
-    this.unorderedBindings = new Set
+    this.eventOptions = this.convertEventOptions(eventOptions)
+    this.unorderedBindings = new Set()
   }
 
   connect() {
-    this.eventTarget.addEventListener(this.eventName, this, false)
+    this.eventTarget.addEventListener(this.eventName, this, this.eventOptions)
```

Comment:
We’ll also need to pass `this.eventOptions` as the third argument to `removeEventListener()` in `disconnect()` below.